ib_insync/decoder.py

```diff
@@ -60,6 +60,8 @@ def __init__(self, wrapper: Wrapper, serverVersion: int):
                 'scannerParameters', [str]),
             20: self.scannerData,
             21: self.tickOptionComputation,
+            32: self.wrap('tickString', [str]),
+            33: self.wrap('tickString', [str]),
```

Comment:
The numbers 32 and 33 are not message types (they're tick types) and these two lines can be removed.